### PR TITLE
[TeamQuest] Add index view and template

### PR DESCRIPTION
**Pull request by razvan3895**

### CODE REVIEW

File: wouso/games/teamquest/views.py [hunk 1]
```diff
@@ -12,6 +12,68 @@
...
+        return context
+
+    def post(self, request, **kwargs):
+        quest = TeamQuestGame.get_current()
+        quest_user = request.user.get_profile().get_extension(TeamQuestUser)
+        status = TeamQuestStatus.objects.get(quest=quest, group=quest_user.group)
+
+        for level in status.levels.all():
+            for question in level.questions.all():
+                answer = request.POST.get('form'+str(question.index))
```
Comment (iulianR) [inline wouso/games/teamquest/views.py]: ' + str

File: wouso/games/teamquest/templates/teamquest/index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+		{% if group %}
+
+			{% if status %}
+
+				{% if status.progress < quest.total_points %}
+
+				<br>
+				<h3>
+				<p> You are now adventuring in quest "{{ quest.title }}". </p>
```
Comment (razvand) [inline wouso/games/teamquest/templates/teamquest/index.html]: Use `{% trans message %}` throughout the template file.

File: wouso/games/teamquest/views.py [hunk 3]
```diff
@@ -12,6 +12,68 @@
...
+                        messages.success(request, 'Congratulations! You have finished this quest on position #%d!' % level.level.times)
+                        status.time_finished = datetime.datetime.now()
+                        status.save()
+
+                    else:
+                        question.state = 'A'
+                        question.save()
+
+                        if level.completed:
+                            messages.success(request, 'Congratulations! You have finished this level on position #%d!' % level.level.times_completed)
```
Comment (razvand) [inline wouso/games/teamquest/views.py]: Use `ugettext` for i18n support for messages in Python files.

File: wouso/games/teamquest/views.py [hunk 4]
```diff
@@ -9,10 +9,77 @@
...
+                if question.state == 'A':
+                    messages.error(request, _("Puny human, don't try to cheat!"))
+                    return HttpResponseRedirect(request.META.get('HTTP_REFERER'))
+
+                question.state = 'A'
+                question.save()
+
+                if question.level.questions.all().count() == 1:
+                    messages.success(request, _('Congratulations! You have finished this quest on position #%(tc)d!') % {'tc': level.level.times_completed})
+                    status.time_finished = datetime.datetime.now()
```
Comment (iulianR) [inline wouso/games/teamquest/views.py]: Create a new method that sets a TeamQuest as finished for a team.

**Pull request merged**